.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,72 @@
+name: Deploy
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@master
+
+      - name: start deployment
+        uses: bobheadxi/deployments@master
+        id: deployment
+        with:
+          step: start
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is this `secrets` object defined in Dockerhub or Digitalocean?

Reply:
These secrets are [defined in GitHub](https://github.com/codebuddies/backend/settings/secrets). You enter them once but can't see the values again. They get presented to your actions for use during execution.

Reply:
Gotcha, that makes sense, thanks!